pycheribuild/projects/disk_image.py

```diff
@@ -198,16 +198,16 @@ def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         # make use of the mtree file created by make installworld
         # this means we can create a disk image without root privilege
-        self.manifest_file = None  # type: Optional[Path]
-        self.extra_files: "list[Path]" = []
+        self.manifest_file: Optional[Path] = None
+        self.extra_files: List[Path] = []
```

Comment:
I would keep this then the import can be omitted.

Reply:
If we really want to omit the import and still remove the `# type: ` comments perhaps this can become
```
        self.manifest_file: "Path|None" = None
        self.extra_files: "list[Path]" = []
```
Which I think is better because it eventually can be enabled without the quotes in more recent python versions.

Reply:
Oh perhaps you were only referring to the `List` import, I see.